3Dmol/glviewer.js

```diff
@@ -2295,34 +2299,51 @@ $3Dmol.GLViewer = (function() {
             if (options.reps) {
                 reps = options.reps;
             }
-            var mostFrames = this.getFrames();
+            var mostFrames = this.getNumFrames();
             var that = this;
             var currFrame = 0;
             var inc = 1;
             var displayCount = 0;
             var displayMax = mostFrames * reps;
+            var time = new Date();
+            var resolve = function() {
+                that.render();
+                if (++displayCount == displayMax || !that.isAnimated()) {
+                    clearTimeout(intervalID);
+                    decAnim(); 
+                }
+                else {
+                    var newInterval = interval - (new Date().getSeconds() - time.getSeconds())/1000000.0;
```

Comment:
Oh yeah, I was supposed to use getTime and divide by 1000 to get seconds. Instead of multiplying and cancelling out the 1000, I divided again. Fixed the issue.